packages/browser/src/transports/xhr.ts

```diff
@@ -46,6 +46,9 @@ export class XHRTransport extends BaseTransport {
         };
 
         request.open('POST', this.url);
+        Object.keys(this.options.headers).forEach(function(header: string) {
```

Comment:
This will break for undefined headers. Also using `forEach` with an overridden `this` is kinda confusing.

```js
for (const key in this.options.header) {
  request.setRequestHeader(key, this.options.headers[key]);
}
```

This will work for `undefined` as well as is more straightforward to read imo.

Reply:
I'll change to your recommended usage. My Typescript knowledge is incredibly bare.